setup.py

```diff
@@ -13,7 +13,7 @@
         'python-jose==3.*'
     ],
     extras_require={
-        "cryptography": ['cryptography>=3.*']
+        "cryptography": ['cryptography>=3.4.8']
```

Comment:
I think this might allow upgrades past our current version. We should be able to do something like this as well: 
```suggestion
        "cryptography": ['cryptography==3.*']
```